frontend/pages/hosts/details/DeviceUserPage/EnrollMdmModal/EnrollMdmModal.tsx

```diff
@@ -122,11 +124,45 @@ const ManualEnrollMdmModal = ({
     );
   };
 
+  const renderAutoModalContent = () => {
+    return (
+      <div>
+        <p className={`${baseClass}__description`}>
+          To turn on MDM, Apple Inc. requires that you install a profile.
+        </p>
+        <ol>
+          <li>
+            From the Apple menu in the top left corner of your screen, select{" "}
+            <b>System Settings</b> or <b>System Preferences</b>.
+          </li>
+          <li>
+            In the search bar, type “Profiles.” Select <b>Profiles</b>, find and
+            select <b>Enrollment Profile</b>, and select <b>Install</b>.
+          </li>
+          <li>
+            Enter your password, and select <b>Enroll</b>.
+          </li>
+          <li>
+            Close this window and select <b>Refetch</b> on your My device page
+            to tell your organization that MDM is on.
+          </li>
+        </ol>
+        <div className="modal-cta-wrap">
+          <Button type="button" onClick={onCancel} variant="brand">
+            Done
+          </Button>
+        </div>
+      </div>
+    );
+  };
+
   return (
     <Modal title="Turn on MDM" onExit={onCancel} className={baseClass}>
-      {renderModalContent()}
+      {mdmEnrollmentType === "manual"
```

Comment:
Can we make manual be the default and auto be only if it's set to auto? That way if there's any situation they get to this modal in the future and it's not auto, it'll show manual instructions

Reply:
Does line DeviceUserPage line 266-276 sufficiently do that?

Reply:
I meant so we can switch this 

``` 
{mdmEnrollmentType === "manual"
        ? renderManualModalContent()
        : renderAutoModalContent()}
``` 

to be

```
      {mdmEnrollmentType === "auto"
        ? renderAutoModalContent()
        : renderManualModalContent()}
```

idk